Library/Formula/frescobaldi.rb

```diff
@@ -34,17 +38,21 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
-
-    if build.with? "launcher"
-      system "python", "macosx/mac-app.py", "--force", "--version",  version, "--script", libexec/"bin/frescobaldi"
-      inreplace "dist/Frescobaldi.app/Contents/Resources/__boot__.py",
-                "_path_inject(['#{libexec}/bin'])",
-                "_path_inject(['#{libexec}/bin', '#{libexec}/lib/python2.7/site-packages', '#{libexec}/vendor/lib/python2.7/site-packages'])"
-      prefix.install "dist/Frescobaldi.app"
-    end
   end
 
   test do
     system bin/"frescobaldi", "--version"
   end
+
+  def caveats
+    s = <<-EOS.undent
+      By default, a splash screen is shown on startup; this causes the main
+      window not to show until the application icon on the dock is clicked
+      (Cmd-Tab application switching does not appear to work). You may
+      want to disable the splash screen in the preferences to solve this
+      issue. See:
+        https://github.com/wbsoft/frescobaldi/issues/428
+    EOS
+    return s
```

Comment:
Don't need the `return` or the `s` declaration; in Ruby the last value is implicitly returned from any function.